src/v1/text_to_speech_client.ts

```diff
@@ -193,25 +232,51 @@ class TextToSpeechClient {
     return ['https://www.googleapis.com/auth/cloud-platform'];
   }
 
+  getProjectId(): Promise<string>;
+  getProjectId(callback: Callback<string, undefined, undefined>): void;
   /**
    * Return the project ID used by this class.
    * @param {function(Error, string)} callback - the callback to
    *   be called with the current project Id.
    */
-  getProjectId(callback) {
-    return this.auth.getProjectId(callback);
+  getProjectId(
+    callback?: Callback<string, undefined, undefined>
+  ): Promise<string> | void {
+    if (callback) {
+      this.auth.getProjectId(callback);
+      return;
+    }
+    return this.auth.getProjectId();
   }
 
   // -------------------
   // -- Service calls --
   // -------------------
-
+  listVoices(
+    request: protosTypes.google.cloud.texttospeech.v1.IListVoicesRequest,
+    options?: gax.CallOptions
+  ): Promise<
+    [
+      protosTypes.google.cloud.texttospeech.v1.IListVoicesResponse,
+      protosTypes.google.cloud.texttospeech.v1.IListVoicesRequest | undefined,
+      {} | undefined
+    ]
+  >;
+  listVoices(
+    request: protosTypes.google.cloud.texttospeech.v1.IListVoicesRequest,
+    options: gax.CallOptions,
+    callback: Callback<
+      protosTypes.google.cloud.texttospeech.v1.IListVoicesResponse,
+      protosTypes.google.cloud.texttospeech.v1.IListVoicesRequest | undefined,
+      {} | undefined
+    >
+  ): void;
   /**
    * Returns a list of Voice supported for synthesis.
    *
    * @param {Object} request
    *   The request object that will be sent.
-   * @param {string} [request.languageCode]
+   * @param {string} request.language_code
```

Comment:
nit (for the generator): optional parameters (marked with optional field annotation) should be in square brackets in JSDoc. 

nit (for the generator): they must be in camelCase, not snake_case.

Reply:
https://github.com/googleapis/gapic-generator-typescript/issues/115

Reply:
It does not stop us from merging this since it will be a compatible doc-only change.

Reply:
got it, will work on it in separate pr! Thanks